src/dex/weth/weth.ts

```diff
@@ -181,11 +183,15 @@ export class Weth
     }
 
     if (needWithdraw || destAmount !== '0') {
-      const opType = WethFunctions.withdrawAllWETH;
-      const withdrawWethData = this.simpleSwapHelper.encodeFunctionData(
-        opType,
-        [wethToken],
-      );
+      const opType = WethFunctions.withdraw;
+      // const withdrawWethData = this.simpleSwapHelper.encodeFunctionData(
+      //   opType,
+      //   [wethToken],
+      // );
+
+      const withdrawWethData = this.erc20Interface.encodeFunctionData(opType, [
+        destAmount,
+      ]);
```

Comment:
Why do we need to withdraw specific amount? I assume that it could lead to leaving tokens on Augustus